clients/filesystem-hadoop/src/test/java/com/datastrato/gravitino/filesystem/hadoop/utils/FileSystemUtils.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.filesystem.hadoop.utils;
+
+import com.datastrato.gravitino.filesystem.hadoop.GravitinoVirtualFileSystemConfiguration;
+import com.datastrato.gravitino.filesystem.hadoop.HdfsMiniClusterTestBase;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FSDataOutputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+public class FileSystemUtils {
+  private FileSystemUtils() {}
+
+  public static Path createGvfsPath(
+      String metalakeName, String filesetCatalog, String schema, String fileset, String fileName) {
```

Comment:
Could you give `fileName` a better name?

Reply:
How about the `filePath`?